krita_batch_exporter/Infrastructure.py

```diff
@@ -14,8 +14,33 @@
 
 
 def dataToPIL(wnode):
-    img = wnode.node.projectionPixelData(*wnode.bounds).data()
```

Comment:
There can be a big overhead with the color space conversion in Krita, especially on larger docs and complex exports - imagine a 4k pixels wide document with dozens of layers to export, we're looking at dozens of seconds of extra export time. 
So we should only convert the color space if it's necessary. @razcore-art do you know if there's a way to get the node's color space in the API?

Reply:
> get the node's color space in the API

We can get the model, depth, and the name of the ICC profile with `colorModel`, `colorDepth`, and `colorProfile`, but I don't think we can tell whether the profile is an sRGB one. We can special-case `"sRGB-elle-V2-srgbtrc.icc"` if you'd prefer it that way.